docs/docs/frigate/hardware.md

```diff
@@ -34,20 +34,23 @@ A detector is a device which is optimized for running inferences efficiently to
 
 ### Google Coral TPU
 
-It is strongly recommended to use a Google Coral. A $60 device will outperform $2000 CPU. Frigate should work with any supported Coral device from https://coral.ai
+It is strongly recommended to use a Google Coral unless you want to use larger models to improve detection. A $60 device will outperform $2000 CPU. Frigate should work with any supported Coral device from https://coral.ai
```

Comment:
```suggestion
It is strongly recommended to use a Google Coral. A $60 device will outperform $2000 CPU. Frigate should work with any supported Coral device from https://coral.ai
```

Reply:
Blake has spent a lot of time testing many different models and architectures and so far there hasn't been anything to show that large models are largely more accurate.

Reply:
I don't want to dispute his tests and am happy to remove this. However, in my own testing, YOLOv8 drastically improves detection of true positives and I get almost no false positives anymore. This was very different on the default model. It makes sense, too: According to [the Corals model page](https://coral.ai/models/object-detection/), `SSDLite MobileDet` achieves `32.9% mAP`  while according to [the YOLOv8 performance metrics](https://docs.ultralytics.com/models/yolov8/#performance-metrics), YOLOv8x manages `53.9% mAP`.

Anyway, this is probably not the place to have that discussion.

Reply:
> However, in my own testing,

right, some users do see better results, while other users see worse results and have ditched Nvidia for using coral for example because it had fewer false positives for them. 

We have to be mindful when making claims in the docs how users will read it and what they will expect. We can not guarantee or even feel confident that getting a GPU and running a larger model will make their detection much better and reduce false positives because we don't have enough information to know that that is true across the board.

Reply:
I think at the very least users should perhaps be a bit more encouraged to check different models and see whether they improve detection if current results are subpar? I understand that this is also the one of the primary value propositions of the Frigate+ model (though I also see how this might possibly result in a conflict of interests).

Would you be open to me adding a line or two about encouraging users to experiment with different models in case detection isn't doing too great?

Reply:
That's fair.

Reply:
> I don't want to dispute his tests and am happy to remove this. However, in my own testing, YOLOv8 drastically improves detection of true positives and I get almost no false positives anymore. This was very different on the default model. It makes sense, too: According to [the Corals model page](https://coral.ai/models/object-detection/), `SSDLite MobileDet` achieves `32.9% mAP` while according to [the YOLOv8 performance metrics](https://docs.ultralytics.com/models/yolov8/#performance-metrics), `YOLOv8x` manages `53.9% mAP`.

I have the same experience. 
Have been running coral for a few months and had shitloads of false positives.
Then went to yolov7-320.trt for a few months, excellent detection and very few false positives.
Now my GPU rig died and I had to go to coral again 😭

Reply:
@jmtatsch Well, you could use YOLOv8 with OpenVINO as per these new docs. :P